Dockerfile

```diff
@@ -1,22 +1,45 @@
-FROM python:3.7-slim-stretch
+FROM node:current-alpine AS NODEBUILD
+# copy directory to /opt and work inside it
+WORKDIR /opt
```

Comment:
Not sure what else is in `opt` or ever will be there. How about using a more specific folder? Can reuse `/etc/linkding` as in all the other stages.

Reply:
`/opt` is empty unless it's filled by the user, feel free to change the approrpaite paths to `/etc/linkding`, `/app`, or even `/workdir`.

Reply:
Changed to /etc/linkding.